### PR TITLE
demo commit

### PR DESCRIPTION
**Pull request by spyros-papavlasopoulos**
added some helper classes

### CODE REVIEW

File: _v2_helpers.scss [hunk 1]
```diff
@@ -151,3 +151,89 @@ $staticURL: $remotehost + '/assets/images/marketing';
...
   background: $colorLightGrey;
 }
+
+//-------------------------------------
+// helper classes
+//-------------------------------------
+
+/* font-style */
+.h1 {
+  @include media($breakTabletLandscape) {font-size: 3em !important;}
```
Comment (felixjung) [inline _v2_helpers.scss]: no need for the `;`, right?

File: _v2_helpers.scss [hunk 2]
```diff
@@ -151,3 +151,89 @@ $staticURL: $remotehost + '/assets/images/marketing';
...
+  font-weight: 300;
+  line-height: 1.3;
+}
+.h2 {
+  @include media($breakTabletLandscape) {font-size: 1.6em !important;}
```
Comment (felixjung) [inline _v2_helpers.scss]: Same as above.

File: _v2_helpers.scss [hunk 3]
```diff
@@ -151,3 +151,89 @@ $staticURL: $remotehost + '/assets/images/marketing';
...
+
+
+//-------------------------------------
+// mixins
+//-------------------------------------
+/* apply ellipsis effect on multiple lines */
+@mixin ellipsis-multiple($fontsize, $lineheight, $visiblelines) {
+  display: block; /* Fallback for non-webkit */
+  display: -webkit-box;
+  //height: $font-size*$line-height*$lines-to-show; /* Fallback for non-webkit  */
```
Comment (felixjung) [inline _v2_helpers.scss]: Why comment this out? Does the fallback not work?

File: _v2_helpers.scss (hunk 1 above)
Comment (nilsborchers) [inline _v2_helpers.scss]: Why are we using !important? That's usually not a good idea and should not be necessary

File: _v2_helpers.scss (hunk 2 above)
Comment (nilsborchers) [inline _v2_helpers.scss]: Yep same

**Comment by nilsborchers** [comment]
Ehm guys, I know I'm just gonna make myself unpopular again, but I'll just ask anyway:
What is this about?
Are switching from doing stuff in css to a more bootstrap (we have classes and distribute them all over the markup) kind of way?
Because if so, this is a huge change and will (since these are the GLOBAL styles) affect not only the marketing website.
@mlent what do you think about this?

**Comment by mlent** [comment]
What Nils said.

These styles are shared between projects, and while I agree that our current CSS "practices" are not optimal for the most part (e.g. the fact that a lot of the CSS is just DOM mirroring) -- let's talk about how we actually want to move forward, thinking about all the web components that rely on these styles, establish some best practices, and enforce them on one another.

I say we do that once the static site is launched -- then we'll have some time to think about how to optimize it to the moon :)

**Pull request closed without merging**